bubble-wrap-style.yaml

```diff
@@ -4021,6 +4161,14 @@ layers:
                 mapzen_icon_library:
                     sprite: function() { return feature.religion; }
                     sprite_default: place_of_worship
+                    
+        generator:
+            filter: { kind: generator }
+            draw:
+                mapzen_icon_library:
+                    sprite: function() { return 'generator_' & feature.kind_detail; }
```

Comment:
I think you want `'generator_' + feature.kind_detail` here. You don't want `'generator_' & feature.kind_detail` (single `&` is a bitwise numeric operator) or `'generator_' && feature.kind_detail` (which would always return just `generator_`).